packages/shared/src/components/feeds/MobileFeedActions.tsx

```diff
@@ -38,6 +43,15 @@ export function MobileFeedActions(): ReactElement {
         <Divider className="bg-border-subtlest-tertiary" vertical />
         <MyFeedHeading
           onOpenFeedFilters={() => {
+            if (isEnrolledNotPlus) {
+              openModal({
+                type: LazyModal.AdvancedCustomFeedSoon,
+                props: {},
+              });
+
+              return;
+            }
```

Comment:
Might be better to just show entry point when users click on "feed settings" on old feeds and when enrolled in plus